fastdds_python/CMakeLists.txt

```
@@ -26,6 +26,20 @@ endif()
 
 project(fastdds_python VERSION 2.4.0)
 
+if(NOT WIN32)
+    if(SHARED_LIBRARY_SUFFIX)
+        set(CMAKE_SHARED_LIBRARY_SUFFIX ".${SHARED_LIBRARY_SUFFIX}")
+    else()
+
+        # Default values for shared library suffix
+        if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
+            set(CMAKE_SHARED_LIBRARY_SUFFIX ".so")
+        elseif(${CMAKE_SYSTEM_NAME} STREQUAL "Darwin")
+            set(CMAKE_SHARED_LIBRARY_SUFFIX ".so")
+        endif()
+    endif()
+endif()
```

We decided to always change the value. No need of a user's cmake option.

```suggestion
if(NOT WIN32)
    # Default values for shared library suffix in MacOS
    if(${CMAKE_SYSTEM_NAME} STREQUAL "Darwin")
        set(CMAKE_SHARED_LIBRARY_SUFFIX ".so")
    endif()
endif()
```

---

Done in 36d257a8651b9f1640b852cc8e2ea11817ddc068